learn/sherif/main_test/src/main/java/frc/robot/Constants.java

```diff
@@ -16,8 +16,22 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-    public static final int FRONT_MOTOR = 0;
-    public static final int BACK_MOTOR = 1;
-    public static final int CONTROLLER = 10;
-    public static final int LEFT_STICK_BUTTON = 1;
+    // Drive Motors
+    public static final int DRIVE_LEFT_MOTOR_A = 0;
+    public static final int DRIVE_LEFT_MOTOR_B = 1;
+    public static final int DRIVE_LEFT_MOTOR_C = 2;
+
+    public static final int DRIVE_RIGHT_MOTOR_A = 20;
+    public static final int DRIVE_RIGHT_MOTOR_B = 21;
+    public static final int DRIVE_RIGHT_MOTOR_C = 22;
+
+    // Elevator ID's
+    public static final int THROAT_FRONT = 5;
+    public static final int THROAT_BACK = 6;
+    public static final int ELEVATOR_FRONT = 7;
+    public static final int ELEVATOR_BACK = 8;
+    public static final int INTAKE_SOLENOID = 10;
```

Comment:
There will also need to be an ID for the INTAKE motor (which is separate from solenoid which is used for the pneumatics)